scheduler/config/config.go

```diff
@@ -39,19 +39,78 @@ type Config struct {
 
 func New() *Config {
 	return &Config{
-		Scheduler: NewDefaultSchedulerConfig(),
-		Server:    NewDefaultServerConfig(),
-		DynConfig: NewDefaultDynConfig(),
-		Manager:   NewDefaultManagerConfig(),
-		Host:      NewHostConfig(),
-		Job:       NewDefaultJobConfig(),
-	}
-}
-
-func NewHostConfig() *HostConfig {
-	return &HostConfig{
-		Location: "",
-		IDC:      "",
+		Scheduler: &SchedulerConfig{
+			DisableCDN:           false,
+			ABTest:               false,
+			AScheduler:           "",
+			BScheduler:           "",
+			WorkerNum:            runtime.GOMAXPROCS(0),
+			BackSourceCount:      3,
+			AccessWindow:         3 * time.Minute,
+			CandidateParentCount: 10,
+			Scheduler:            "basic",
+			CDNLoad:              100,
+			ClientLoad:           10,
+			OpenMonitor:          true,
+			GC: &GCConfig{
+				PeerGCInterval: 5 * time.Minute,
+				TaskGCInterval: 5 * time.Minute,
+				PeerTTL:        10 * time.Minute,
+				PeerTTI:        3 * time.Minute,
+				TaskTTL:        10 * time.Minute,
+				TaskTTI:        3 * time.Minute,
+			},
+		},
+		Server: &ServerConfig{
+			IP:   iputils.HostIP,
+			Host: iputils.HostName,
+			Port: 8002,
+		},
+		DynConfig: &DynConfig{
+			Type:       dc.LocalSourceType,
+			ExpireTime: 30 * time.Second,
+			CDNDirPath: "",
+			Data: &DynconfigData{
+				CDNs: []*CDN{
+					{
+						HostName:      "localhost",
+						IP:            "127.0.0.1",
+						Port:          8003,
+						DownloadPort:  8001,
+						SecurityGroup: "",
+						Location:      "",
+						IDC:           "",
+						NetTopology:   "",
+					},
+				},
+			},
+		},
+		Manager: &ManagerConfig{
+			Addr:               "",
+			SchedulerClusterID: 0,
+			KeepAlive: KeepAliveConfig{
+				Interval:         5 * time.Second,
+				RetryMaxAttempts: 100000000,
+				RetryInitBackOff: 5,
+				RetryMaxBackOff:  10,
+			},
+		},
+		Host: &HostConfig{
+			Location: "",
+			IDC:      "",
+		},
+		Job: &JobConfig{
+			GlobalWorkerNum:    10,
+			SchedulerWorkerNum: 10,
+			LocalWorkerNum:     10,
+			Redis: &RedisConfig{
+				Host:      "",
+				Port:      6379,
+				Password:  "",
+				BrokerDB:  1,
+				BackendDB: 2,
+			},
+		},
 	}
```

Comment:
What are the benefits of the configuration items taken together

Reply:
Look at other module config.